frontend/src/components/common/form/Input.tsx

```diff
@@ -94,30 +94,29 @@ export const Input: React.FC<InputProps> = ({
           {!!tooltip && <TooltipIcon toolTipId={`${field}-tooltip`} toolTip={tooltip} />}
         </Form.Label>
       )}
-      <span className={className}>
```

Comment:
As far as I can see there is no reason for this span to be here, I've had a look through the pages affected by removing this and there doesn't appear to be a difference. Removing this:
simplifies the dom
makes findByLabelText work